working/templates/package-project/GettingStarted.md

```diff
@@ -0,0 +1,154 @@
+
+### Other DataMiner Projects in the Same Solution
+
+Every **Skyline.DataMiner.SDK** project, except other DataMiner Package Projects, will by default be included within the `.dmapp` created by this project.  
+You can customize this behavior using the **PackageContent/ProjectReferences.xml** file. This allows you to add filters to include or exclude projects as needed.
+
+### Adding Content from the Catalog
+
+You can reference and include additional content from the catalog using the **PackageContent/CatalogReferences.xml** file provided in this project.
+
+### Importing from DataMiner
+
+You can import specific items directly from a DataMiner agent:  
+
+1. Connect to an agent via **Extensions > DIS > DMA > Connect**.
+2. Once connected, you can import specific DataMiner artifacts.
+3. Navigate to folders such as **PackageContent/Dashboards** or **PackageContent/LowCodeApps**, right-click, select **Add**, and choose **Import DataMiner Dashboard/LowCodeApp** or the equivalent.
+
+## Execute Additional Code on Installation
+
+Open the **$SCRIPTNAME$.cs** file to write custom installation code. Common actions include creating elements, services, or views.
+
+**Quick Tip:** Type `clGetDms` in the `.cs` file and press **Tab** twice to insert a snippet that gives you access to the **IDms** classes, making DataMiner manipulation easier.
+
+## Does Your Installation Code Need Configuration Files?
+
+You can add configuration files (e.g., `.json`, `.xml`) to the **SetupContent** folder, which can be accessed during installation.
+
+Access them in your code using:
+```csharp
+string setupContentPath = installer.GetSetupContentDirectory();
+```
+
+<!--#else-->
+## Enabling the Creation of a DataMiner Application Package
+
+**OOPS!** This project is not configured to generate a `.dmapp` file.  
+If you intended to create a `.dmapp` file, you may have set up the project incorrectly.
+
+Please consider the following options:
+1. Remove this project and create a new DataMiner Application Project with `.dmapp` generation enabled.
+2. If you don’t require `.dmapp` creation, review your project setup to ensure it aligns with your goals.
+<!--#endif-->
+
+<!--#if (IsCatalogNoCICD)-->
+## Publishing to the Catalog
+
+This project was created with support for publishing to the DataMiner catalog.  
+You can publish your artifact manually through Visual Studio or by setting up a CI/CD workflow.
+
+### Manual Publishing
+
+1. Obtain an **Organization Key** from [admin.dataminer.services](https://admin.dataminer.services/) with the following scopes:
+   - **Register catalog items**
+   - **Read catalog items**
+
+2. Securely store the key using Visual Studio User Secrets:
+   - Right-click the project and select **Manage User Secrets**.
+   - Add the key in the following format:
+
+```json
+{
+  "skyline": {
+    "sdk": {
+      "catalogpublishtoken": "MyKeyHere"
+    }
+  }
+}
+```
```

Comment:
This example should be indented at the same level as the text of line 76; otherwise you risk issues with the list numbering.

Reply:
Applied